crates/sui-graphql-rpc/src/consistency.rs

```diff
@@ -147,6 +147,7 @@ pub(crate) fn build_objects_query(
     // Similar to the snapshot query, construct the filtered inner query for the history table.
     let mut history_objs_inner = query!("SELECT * FROM objects_history");
     history_objs_inner = filter_fn(history_objs_inner);
+    history_objs_inner = filter!(history_objs_inner, "object_status = 0");
```

Comment:
sry that i missed it here and thx for the fix, is there an easy way to generate a sample query here so that I can verify it on psql?

Reply:
You didn't miss it in this case -- I've just moved the filter up into a central place, rather than applying at the end to each query, because it was ambiguous whether the `object_status` query would apply just to the historical candidates, the newer values or both (and we only want it to apply to the historical candidates).

If you run the GraphQL service with debug logs enabled, you can see every query that gets run. I ran the tests as follows for example:

```
sui$ RUST_LOG="info,sui_graphql_rpc::data::pg=debug" \
  cargo nextest run -p sui-graphql-e2e-tests --no-capture \
  -- call/dynamic_fields.move
```